vaadin-grid-flow-parent/vaadin-grid-flow/src/main/java/com/vaadin/flow/component/grid/Grid.java

```diff
@@ -1028,6 +1031,39 @@ public Column<T> setClassNameGenerator(
             return this;
         }
 
+        /**
+         * Sets the function that is used for generating tooltip text for cells
+         * in this column. Returning {@code null} from the generator results in
+         * no tooltip being set.
+         *
+         * @param tooltipTextGenerator
+         *            the tooltip text generator to set, not {@code null}
+         * @return this column
+         * @throws NullPointerException
+         *             if {@code classNameGenerator} is {@code null}
+         */
+        public Column<T> setTooltipTextGenerator(
+                SerializableFunction<T, String> tooltipTextGenerator) {
+            if (!getGrid().getElement().getChildren().anyMatch(
```

Comment:
Is the typical Objects.requireNonNull missing?

Reply:
Added